tests/integration/test_iv_values_usgs.py

```diff
@@ -0,0 +1,80 @@
+from typing import Iterator
+
+import pytest
+from pydantic import ValidationError
+
+from basin3d.core.schema.enum import ResultQualityEnum, TimeFrequencyEnum
+from basin3d.synthesis import register
+
+
+@pytest.mark.integration
+def test_measurement_timeseries_tvp_observations_usgs_iv():
+    """ Test USGS Timeseries data query"""
+
+    synthesizer = register(['basin3d.plugins.usgs.USGSDataSourcePlugin'])
+
+    query0 = {
+        "monitoring_features": ["USGS-09110990", "USGS-09111250"],
+        "observed_property_variables": [],
+        "start_date": "2020-04-01",
+        "end_date": "2020-04-30",
+        "aggregation_duration": TimeFrequencyEnum.NONE,
+        "results_quality": ResultQualityEnum.VALIDATED
+    }
+
+    with pytest.raises(ValidationError):
+        synthesizer.measurement_timeseries_tvp_observations(**query0)
+
+    query1 = {
+        "monitoring_features": ["USGS-09110990", "USGS-09111250"],
+        "observed_property_variables": ["RDC"],
+        "start_date": "2020-04-01",
+        "end_date": "2020-04-30",
+        "aggregation_duration": TimeFrequencyEnum.NONE,
+        "results_quality": ResultQualityEnum.VALIDATED
+    }
+    measurement_timeseries_tvp_observations = synthesizer.measurement_timeseries_tvp_observations(**query1)
+    if isinstance(measurement_timeseries_tvp_observations, Iterator):
+        count = 0
+        for timeseries in measurement_timeseries_tvp_observations:
+            print(timeseries.to_json())
+            count += 1
+
+        assert count == 2
+    else:
+        pytest.fail("Returned object must be iterator")
+
+    query2 = {
+        "monitoring_features": ["USGS-09110990", "USGS-09111250"],
+        "observed_property_variables": ["RDC"],
+        "start_date": "2020-04-01",
+        "end_date": "2020-04-30",
+        "aggregation_duration": TimeFrequencyEnum.NONE,
+        "statistic": ["MEAN"],
+        "results_quality": ResultQualityEnum.VALIDATED
+    }
+    measurement_timeseries_tvp_observations = synthesizer.measurement_timeseries_tvp_observations(**query2)
+    if isinstance(measurement_timeseries_tvp_observations, Iterator):
+        count = 0
+        for timeseries in measurement_timeseries_tvp_observations:
+            print(timeseries.to_json())
+            count += 1
```

Comment:
Add an assert that the MEAN statistic is ignored.

Reply:
I still would like to see an assert that the MEAN statistic does not show up in the metadata. Additionally, you could check that the message from usgs.py new line 133 shows up in the synthesis messages.

Reply:
@dschristianson 

- I do see the messages from usgs.py new line 133 in synthesis messages but not in `measurement_timeseries_tvp_observations` SynthesisResponse.messages since at that time there will be no stored messages yet. 
- I didn't see the MEAN statistic in the metadata without the assert statements I added below, but was there a place you did see it?

 ```
   if isinstance(measurement_timeseries_tvp_observations, Iterator):
        count = 0
        for timeseries in measurement_timeseries_tvp_observations:
            timeseries.to_json()
            count += 1
            # checking to make sure the query statistic is not the statistic in timeseries data,
            # and checking to make sure that when aggregation duration is NONE the statistic in timeseries data has the value 'NOT_SUPPORTED'
            assert query2['statistic'][0] != timeseries.statistic
            assert timeseries.statistic == 'NOT_SUPPORTED'
        assert count == 2
    else:
        pytest.fail("Returned object must be iterator")
```

Reply:
Apologies if I missed these statements in the last review. I was looking for something like what you have: `assert timeseries.statistic == 'NOT_SUPPORTED'`

WRT synthesis messages, I think you should be able to assert to them. See example code in test/test_plugin_usgs.py, lines 115-120.